### ISSUE TITLE
number of samples

### ISSUE THREAD

**Issue by jfilter** [tblock/10kGNAD#1]
I have 9245 samples in the training set and 1028 in the test set. Is this correct? I was wondering because they account to more than 10k samples.

Thanks for you work. It's a good idea to a have an easy-to-use data-set for goofing around.

**Comment by jfilter** [comment]
There are correct. You write more about the samples on your website: https://tblock.github.io/10kGNAD/
